test/sandbox/test/collection_test.rb

```diff
@@ -17,5 +26,10 @@ def test_collection_has_a_size
     def test_is_a_collection_of_view_components
       assert_equal [ProductComponent], @collection.map(&:class).uniq
     end
+
+    def test_supports_components_with_keyword_args
+      collection = ComponentWithKeywordArgs.with_collection(@products)
+      render_inline(collection)
```

Comment:
```suggestion
      render_inline(ComponentWithKeywordArgs.with_collection(@products))
```

Can you add an assertion that something is rendered correctly?

Reply:
Done!